content/community/nav.yml

```
@@ -29,7 +29,7 @@
     - id: starter-kits
       title: أدوات البداية
     - id: routing
-      title: المسارات
+      title: التوجيه
```

أعتقد أن لا نغيرها فالتوجية قد يقصد به 
direction

---

@iRayan7

---

> أعتقد أن لا نغيرها فالتوجية قد يقصد به
> direction

IMO, the translation is a bit confusing, but I think that the current convention & what @iRayan7 suggested is more correct, since a `router` in Arabic is `المُسَيّر` or `المُوَجّه`, since it's responsible of managing "the choice of the direction (the route)" so it's a  `مُوجّه`. The word `مسار` is closer to `path` (and it's used like that in the majority of the Arabic translations).

TLDR; I support the current convention, we should keep `المُوَجّه` as a translation for `router`.

```suggestion
      title: الموَجّهات
```
Or (as it is :smile: )
```suggestion
      title: التوجيه
```

---

I agree with @Fcmam5. Since this page is for routing libraries, and these libraries are used for `توجيه` the user.

Also, `موجهات` is closer to `routers`. `توجيه` is `routing` 👍